### PR TITLE
Add link explaining trendlines changes

### PR DESCRIPTION
**Pull request by selvach**
Adds a link onto the trendline card that links to a blog explaining the switch to 14-day sma. 

Screenshot 2021-03-08 at 21 16 11

### CODE REVIEW

File: src/components/Cards/EstimatedCase/TrendlineCard.tsx [hunk 1]
```diff
@@ -44,9 +45,19 @@ export const TrendlineCard: React.FC<Props> = ({ ctaOnPress = () => null, isShar
         <Text textClass="h4" rhythm={8}>
           {i18n.t('explore-trend-line.active-covid-cases')} {localTrendline?.name}
         </Text>
-        <Text textClass="pSmallLight" rhythm={32} colorPalette="uiDark" colorShade="dark" inverted>
-          {i18n.t('explore-trend-line.evolution-of')}
-        </Text>
+        {/*<Text textClass="pSmallLight" rhythm={8} colorPalette="uiDark" colorShade="dark" inverted>*/}
```
Comment (bionara) [inline src/components/Cards/EstimatedCase/TrendlineCard.tsx:48]: Remove commented out code, or add comment for why it's left

**Review by bionara — DISMISSED** [review]
Approved with 2 minor comments

File: assets/lang/sv-SE.json [hunk 2]
```diff
@@ -1155,7 +1155,8 @@
     "cta": "",
     "active-covid-cases": "",
     "evolution-of": "",
-    "active-cases-in-your-area": ""
```
[the remark this answers is not in the document]
Comment (selvach) [reply assets/lang/sv-SE.json:1158]: There are no trendlines in Sweden! They publish their data on this website: https://csss-resultat.shinyapps.io/csss_dashboard/

It's pretty cool!

[the remark this answers is not in the document]
Comment (bionara) [reply assets/lang/sv-SE.json:1158]: awesome!

**Comment by selvach** [comment]
Updated copy

File: src/components/Cards/EstimatedCase/TrendlineCard.tsx [hunk 3]
```diff
@@ -44,9 +45,16 @@ export const TrendlineCard: React.FC<Props> = ({ ctaOnPress = () => null, isShar
...
-        <Text textClass="pSmallLight" rhythm={32} colorPalette="uiDark" colorShade="dark" inverted>
-          {i18n.t('explore-trend-line.evolution-of')}
-        </Text>
+
+        <View style={{ paddingBottom: 8 }}>
+          <Link
+            color={colors.linkBlue}
+            iconName="info"
+            linkText={i18n.t('explore-trend-line.trendline-change')}
+            onPress={() => Linking.openURL('https://covid.joinzoe.com/post/covid-rates-trends-changing-near-you')}
```
Comment (bionara) [inline src/components/Cards/EstimatedCase/TrendlineCard.tsx:54]: Do we wanna use openWebLink instead - our in-house one that comes with tracking etc?

**Review by bionara — APPROVED** [review]
approved with Q about openURL

File: src/components/Cards/EstimatedCase/TrendlineCard.tsx (hunk 3 above)
Comment (selvach) [reply src/components/Cards/EstimatedCase/TrendlineCard.tsx:54]: Great shout!

**Review by bionara — APPROVED** [review]

**Pull request merged**